design/kanister-kopia-integration.md

```diff
@@ -0,0 +1,54 @@
+# Integrating Kopia with Kanister
+
+This document proposes all the changes required within Kanister to better support the use of Kopia backup/restore tool.
+
+## Introducing Kopia
+
+[Kopia](https://kopia.io/) is a powerful, cross-platform tool for managing encrypted backups in the cloud.
+It provides fast and secure backups, using compression, data deduplication and client-side end-to-end encryption.
+It supports a variety of backup storage targets, including object stores, which allows users to choose the storage provider that better addresses their needs.
+It is a lock-free system that allows for concurrent multi-client operations including garbage collection.
+
+## Goal
+
+Kopia can focus on its core data export and transformation as part of various Kanister functions.
+While, Kanister continues to execute these functions from it's application-centric data protection workflows called Blueprints.
```

Comment:
I am not sure what this means and how this benefits Kanister users and downstream applications. I think this section should describe what problems we are trying to solve, and what is lacking in the current approach.